runtime/lua/vim/_editor.lua

```diff
@@ -810,6 +810,44 @@ function vim.deprecate(name, alternative, version, plugin, backtrace)
   end
 end
 
+--- Create builtin mappings (incl. menus).
+--- Called once on startup.
+function vim._init_default_mappings()
+  -- mappings
+
+  --@private
+  local function map(mode, lhs, rhs)
+    vim.api.nvim_set_keymap(mode, lhs, rhs, { noremap = true, desc = 'Nvim builtin' })
+  end
+
+  map('n',  'Y',      'y$')
+  -- Use normal! <C-L> to prevent inserting raw <C-L> when using i_<C-O>
+  -- See https://github.com/neovim/neovim/issues/17473
+  map('n',  '<C-L>',  '<Cmd>nohlsearch<Bar>diffupdate<Bar>normal! <C-L><CR>')
+  map('i',  '<C-U>',  '<C-G>u<C-U>')
+  map('i',  '<C-W>',  '<C-G>u<C-W>')
+  map('x',  '*',      'y/\\V<C-R>"<CR>')
```

Comment:
- is this really equivalent? the C string was `"y/\\\\V<C-R>\"<CR>"`. In Lua, single-quotes still need similar escaping. To avoid escaping at the Lua layer, use `[[...]]`.
- similar for the one below

```suggestion
  map('x',  '*',      [[y/\\V<C-R>"<CR>]])
```